script/hassfest/__main__.py

```diff
@@ -214,7 +214,7 @@ def main() -> int:
         return 0
 
     if config.action == "generate":
-        print("Found errors. Generating files canceled.")
+        print("Found errors. Generating files cancelled.")
```

Comment:
Both spellings are acceptable

https://www.merriam-webster.com/grammar/canceled-or-cancelled

Reply:
Ok! 

My editor underlined it in red, probably because it was on british mode.

I never knew this was spelled the other way in USA. Will close the PR.